target/linux/ipq40xx/base-files/etc/board.d/02_network

```diff
@@ -12,7 +12,24 @@ ipq40xx_setup_interfaces()
 
 	case "$board" in
 	8dev,habanero-dvk|\
-	8dev,jalapeno|\
+	avm,fritzbox-4040|\
+	asus,rt-ac58u|\
+	linksys,ea6350v3|\
+	zyxel,nbg6617)
+		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3 lan4" "wan"
+		;;
+	8dev,jalapeno)
+		ucidef_set_interfaces_lan_wan "lan" "wan"
+		;;
+	aruba,ap-303)
```

Comment:
here we need to add sxtsq-5ac

Reply:
has it been converted?

Reply:
Yes? I have already the old version without calibration running and its working very good. However, I have no device that is not build somewhere on a roof. Last time I tried without psgmii-id, it broke. But I would like to test it again. I'm unsure if I can try.

Reply:
Well, if plain RGMII broke networking last time it's gonna do it again (That's if it was the PHY mode).
So, it really depends on whether you are ready to go and recover it again.

Reply:
I converted the dts. So we cann add it here:

```diff
diff --git a/target/linux/ipq40xx/base-files/etc/board.d/02_network b/target/linux/ipq40xx/base-files/etc/board.d/02_network
index 226ebb4c34..360966fd18 100644
--- a/target/linux/ipq40xx/base-files/etc/board.d/02_network
+++ b/target/linux/ipq40xx/base-files/etc/board.d/02_network
@@ -21,11 +21,12 @@ ipq40xx_setup_interfaces()
        8dev,jalapeno)
                ucidef_set_interfaces_lan_wan "lan" "wan"
                ;;
-       aruba,ap-303)
+       aruba,ap-303|\
+       mikrotik,sxtsq-5-ac)
                ucidef_set_interface_lan "lan"

```